.run/Remote Debug Cardinal port 40000.run.xml

```diff
@@ -0,0 +1,6 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
This new file is a run configuration for goland. You will see it in the right hand corner. It will allow you to debug cardinal if you activate it with `mage startDebug`